src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -81,22 +76,35 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
               IndexLogEntry.schemaString(schema),
               numBuckets)),
           Content(path.toString, Seq()),
-          Source(SparkPlan(sourcePlanProperties), Seq(Hdfs(sourceDataProperties))),
+          Source(
+            SparkPlan(sourcePlanProperties)),
```

Comment:
can you run "reformat code"? (Did you configure your IDE to use /dev/.scalafmt.conf?)